File: delete_files.py
import os

files = [
    r'b:\tclinic_nhom3\client\src\pages\admin\AdminDoctors.jsx',
    r'b:\tclinic_nhom3\client\src\pages\admin\AdminDoctors.module.css',
    r'b:\tclinic_nhom3\client\src\pages\doctor\DoctorSelfScheduleWeekly.jsx',
    r'b:\tclinic_nhom3\client\src\pages\doctor\DoctorSelfScheduleWeekly.module.css',
    r'b:\tclinic_nhom3\client\src\components\NavigationControls.jsx',
    r'b:\tclinic_nhom3\client\src\components\NavigationControls.module.css'
]

print('Deleting files...')
for file in files:
    try:
        os.remove(file)
        print(f'✓ Deleted: {file}')
    except FileNotFoundError:
        print(f'✗ Not found: {file}')
    except Exception as e:
        print(f'✗ Error deleting {file}: {e}')

print('\nVerifying deletion...')
all_deleted = True
for file in files:
    exists = os.path.exists(file)
    status = '✗ STILL EXISTS' if exists else '✓ Confirmed deleted'
    print(f'{status}: {file}')
    if exists:
        all_deleted = False

print(f'\nResult: All files deleted successfully!' if all_deleted else '\nResult: Some files still exist!')

File: delete_files.js
const fs = require('fs');
const path = require('path');

const files = [
  'b:\\tclinic_nhom3\\client\\src\\pages\\admin\\AdminDoctors.jsx',
  'b:\\tclinic_nhom3\\client\\src\\pages\\admin\\AdminDoctors.module.css',
  'b:\\tclinic_nhom3\\client\\src\\pages\\doctor\\DoctorSelfScheduleWeekly.jsx',
  'b:\\tclinic_nhom3\\client\\src\\pages\\doctor\\DoctorSelfScheduleWeekly.module.css',
  'b:\\tclinic_nhom3\\client\\src\\components\\NavigationControls.jsx',
  'b:\\tclinic_nhom3\\client\\src\\components\\NavigationControls.module.css'
];

console.log('Deleting files using fs.unlinkSync()...\n');

for (const file of files) {
  try {
    if (fs.existsSync(file)) {
      fs.unlinkSync(file);
      console.log(`✓ Deleted: ${file}`);
    } else {
      console.log(`✗ Not found: ${file}`);
    }
  } catch (err) {
    console.log(`✗ Error deleting ${file}: ${err.message}`);
  }
}

console.log('\nVerifying deletion...\n');
let allDeleted = true;
for (const file of files) {
  const exists = fs.existsSync(file);
  const status = exists ? '✗ STILL EXISTS' : '✓ Confirmed deleted';
  console.log(`${status}: ${file}`);
  if (exists) allDeleted = false;
}

console.log(`\n${allDeleted ? 'Result: All files deleted successfully!' : 'Result: Some files still exist!'}`);
